### PR TITLE
Allow reading raw measurement data

### PR DESCRIPTION
**Pull request by dbrgn**
Add two new methods:

- `get_measurement_result_raw`
- `get_partial_measurement_result_raw`

Additionally, improve docs.

### CODE REVIEW

File: src/types.rs [hunk 1]
```diff
@@ -15,9 +15,27 @@ pub struct Measurement {
     pub humidity: Humidity,
 }
 
+/// A combined raw temperature / humidity measurement.
+#[derive(Debug, Clone, PartialEq, Eq)]
```
Comment (rnestler) [inline src/types.rs]: I'd also derive `Copy` since the type has just the size of an `u32`.

File: src/types.rs [hunk 2]
```diff
@@ -15,9 +15,27 @@ pub struct Measurement {
...
+pub struct RawMeasurement {
+    /// The measured temperature (raw value).
+    pub temperature: u16,
+    /// The measured humidity (raw value).
+    pub humidity: u16,
+}
+
+impl Into<Measurement> for RawMeasurement {
+    fn into(self) -> Measurement {
```
Comment (rnestler) [inline src/types.rs]: I'd prefer to implement `From` for`Measurement` and get the generic `Into` implementation.

**Review by rnestler — CHANGES_REQUESTED** [review]
I'd also add a CHANGELOG entry. Also there are some unrelated documentation fixes in the commit.

File: src/types.rs (hunk 1 above)
Comment (dbrgn) [reply src/types.rs]: Good point, I added it to all four measurement types.

**Comment by dbrgn** [comment]
@rnestler issues addressed, unrelated documentation changes were committed straight to master.

**Review by rnestler — APPROVED** [review]

**Comment by dbrgn** [comment]
@rnestler thanks for the review 🙂

**Pull request merged**